milo-libs/merch-card-collection.js

```diff
@@ -0,0 +1,147 @@
+// branch: MWPW-142267 commit: 25320da3fd942c99cfcd7ec563122d0a7e755d9f Mon, 15 Jul 2024 22:23:34 GMT
```

Comment:
I know in milo we should generate from `main`, is that also the case here @3ch023 ? Sorry for the confusion.

cc @npeltier